unit_test/sparse/Test_Sparse_spmv.hpp

```diff
@@ -641,6 +641,7 @@ TEST_F( TestCategory,sparse ## _ ## spmv_struct ## _ ## SCALAR ## _ ## ORDINAL #
   test_spmv_struct_2D<SCALAR,ORDINAL,OFFSET,DEVICE> (250, 200, 3, 3);      \
```

Comment:
Can we have some odd numbers so we don't make any mistakes on dividing ?

Reply:
Sure I will add some variety here